app.js

```diff
@@ -219,44 +228,64 @@ var storeLocationObjectLiteralsArray = [
 ];
 
 // Create  table element.
-createTableElem();
+var newTable = createTableElem();
+newTable.id = 'cool';
 
 
 // Fill in the rest of the table.
+var tRow = createTableRowElem(newTable);
 
 for (var k = 0; k < storeLocationObjectLiteralsArray.length; k++) {
-  
+
   var currentStoreLocationObjectLiteral = storeLocationObjectLiteralsArray[k];
 
-  createTableHeadElem();
-  createTableRowElem();
+  // var tHead = createTableHeadElem(newTable);
 
   if (k === 0) {
-    createTHElem(tableStrings);
+    createTHElem(tableStrings, tRow);
   } else {
-    createTHElem(currentStoreLocationObjectLiteral.storeLocation);
+    createTHElem(currentStoreLocationObjectLiteral.storeLocation, tRow);
   }
 
 }
 
-for (var l = 0; l < currentStoreLocationObjectLiteral.hourlyCookiesPurchasedArray.length; l++) {
+for(var windex = 0; windex < storeLocationObjectLiteralsArray[1].hourlyCookiesPurchasedArray.length; windex++){
 
+  tRow = createTableRowElem(newTable);
+  for (var lindex = 0; lindex < storeLocationObjectLiteralsArray.length; lindex++) {
+    if(lindex === 0){
+      createTDElem('No total yet', tRow);
+    } else {
+      //string magic
+      var data = storeLocationObjectLiteralsArray[lindex].hourlyCookiesPurchasedArray[windex];
+      data = data.substring(0, data.length - 7);
+      var indexOfColon = data.indexOf(':');
+      data = data.substring(indexOfColon + 1, data.length);
 
-  createTableBodyElem();
-  createTableRowElem();
+      createTDElem(data, tRow);
+    }
 
-  // Add <td> entries from the hourly cookies array.
-  createTDElem(currentStoreLocationObjectLiteral.hourlyCookiesPurchasedArray[l]);
-
-  // Within the second for loop a conditional to launch a final result of total
-  if (l === (currentStoreLocationObjectLiteral.hourlyCookiesPurchasedArray.length - 1)) {
-    createStringsForTDElem();
   }
-
 }
 
+// tRow = createTableRowElem(newTable);
```

Comment:
please remove Zombie code

Reply:
NO. I refuse. In fact, I shall add even more of it. I will have a code zombie army. Prepare your defenses.